lab_1/main.py

```diff
@@ -12,7 +12,16 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
```

Comment:
There should be checks for input arguments. Smth like:
`if not isinstance(input_arg_name, expected_type):`
`do something here`